fs/expose/dokan/__init__.py

```diff
@@ -76,7 +85,9 @@
 from fs.wrapfs import WrapFS
 
 try:
-    import libdokan
+    if six.PY2: import libdokan
+    elif six.PY3: from . import libdokan
```

Comment:
I was trying to take into the consideration of Python4 for no reason... (Which should not exist within a couple of years)